README.md

```diff
@@ -1,76 +1,67 @@
 # knausj_talon
 
-Talon configs for Mac, Windows, and Linux. Very much in progress. This is also intended to work with both Dragon Naturally Speaking and wav2letter.
+Community-maintained [Talon](https://talonvoice.com/) command grammar.
 
-Notes:
+Can be used on its own, but shines when combined with:
 
-- commands are subject to change. We do our best to minimize changes, but we are moving to an [object][verb] standard slowly but surely.
-- @knausj85 makes extensive use of Talon's eye tracking features, so the grammar for certain programs may be much smaller than you may require.
-- The repository was mostly developed with Dragon, so commands are mostly still optimized for that speech engine.
+- [Cursorless](https://www.cursorless.org/) for programming and text editing
+- [Rango](https://github.com/david-tejada/rango) for browser navigation
+- [Other user file sets](https://talon.wiki/talon_user_file_sets/)
 
-## Linux & Mac setup
+## Installation
 
-Clone repo into `~/.talon/user`
+### Prerequisites
 
-```insert code:
-cd ~/.talon/user
-git clone https://github.com/knausj85/knausj_talon knausj_talon
-```
+- [Talon](https://talonvoice.com/)
+- Mac, Windows, or Linux
+- Can work with both Talon's own Conformer speech recognition engine or Talon's Dragon Naturally Speaking integration.
```

Comment:
```suggestion
- Can work with both Talon's built-in Conformer (wav2letter) speech recognition engine (recommended), or Dragon Naturally Speaking (Windows) / Dragon for Mac (although beware that Dragon for Mac is deprecated).
```

I like to mention wav2letter together with Conformer just because Talon sometimes only mentions wav2letter and that's confusing